client/beadledom-client-test/src/test/scala/com/cerner/beadledom/client/ClientServiceSpec.scala

```diff
@@ -66,6 +69,38 @@ class ClientServiceSpec(contextRoot: String, servicePort: Int)
       }
     }
 
+    describe("Pagination") {
+      it("injects pagination links to the response") {
+        val injector = getInjector(List(new ResourceOneModule))
+
+        val paginatedResource = injector.getInstance(classOf[PaginatedClientResource])
+
+        val results : OffsetPaginatedListDto[JsonOne] = paginatedResource.index(0L, 10).body()
+
+        results mustNot be(null)
```

Comment:
If you add 
`results.items().get(0).getHello mustBe "Hello World"`
the test will fail with `java.lang.ClassCastException: java.util.LinkedHashMap cannot be cast to com.cerner.beadledom.client.example.model.JsonOne` that is because Jackson doesn't know how to deserialize the json to java objects [1]. We either need to figure out a way to get TypeReference to be used, or remove the examples from the client since it doesn't work.

[1] https://www.baeldung.com/jackson-collection-array